templates/components/amp/products/options/input-checkbox.html

```diff
@@ -0,0 +1,20 @@
+<div class="form-field" data-product-attribute="input-checkbox">
+    <label class="form-label form-label--alternate form-label--inlineSmall">
+        {{display_name}}:
```

Comment:
🍹 just looks odd to me with the empty line.  maybe one of the following instead?

```
{{display_name}}&colon;
{{#if required}}
  <small>{{lang 'common.required'}}</small>
{{/if}}
```

```
{{display_name}}<span>&colon;</span>
{{#if required}}
  <small>{{lang 'common.required'}}</small>
{{/if}}
```